### ISSUE TITLE
[bug] error: variable 'Column' set but not used [-Werror,-Wunused-but-set-variable]

### ISSUE THREAD

**Issue by agrajag9** [acpica/acpica#849]
https://github.com/acpica/acpica/blob/28fc163aa29e208678d901d98bb9030b775521b3/source/tools/acpisrc/asconvrt.c#L1233

See build log here: https://bz-attachments.freebsd.org/attachment.cgi?id=240459

**Comment by agrajag9** [comment]
```
obj/acpisrc ../../../source/tools/acpisrc/asconvrt.c
../../../source/tools/acpisrc/asconvrt.c:1233:29: error: variable 'Column' set but not used [-Werror,-Wunused-but-set-variable]
    UINT32                  Column = 0;
                            ^
1 error generated.
gmake[2]: *** [../Makefile.rules:20: obj/asconvrt.o] Error 1
```

**Comment by agrajag9** [comment]
Looks like some others as well:
```
../../../source/compiler/aslrestype2s.c:1553:29: error: variable 'PinCount' set but not used [-Werror,-Wunused-but-set-variable]
    UINT32                  PinCount = 0;
                            ^
../../../source/compiler/aslrestype2s.c:1739:29: error: variable 'PinCount' set but not used [-Werror,-Wunused-but-set-variable]
    UINT32                  PinCount = 0;
                            ^
../../../source/compiler/aslrestype2s.c:1939:29: error: variable 'PinCount' set but not used [-Werror,-Wunused-but-set-variable]
    UINT32                  PinCount = 0;
                            ^
3 errors generated.
```

**Comment by sacdintel** [comment]
I believe this issue has been fixed after merging PR #851 so will go ahead and close it. Thanks!
